LCM/Code/App/task.h

```diff
@@ -14,7 +14,7 @@ typedef enum
 } CELL_TYPE;
 
 //#define PINTV
-#define XRV
+//#define XRV
 //#define GTV
 //#define ADV
 
```

Comment:
Defined in `LCM_Light_Control_IO_WS2812_New.csolution.yml` by selecting a device